### PR TITLE
solutik

### PR DESCRIPTION
**Pull request by PyCore23**

### CODE REVIEW

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
fix flake8

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+
+def cinema_visit(
+        customers: list,
```
Comment (Dimosphen1) [inline app/main.py:10]: You could indicate that you expect to obtain a list of dicts for `customers` in type annotation.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Fix flake8 errors and resolve one comment upon the next review request.

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+from app.people.customer import Customer
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+
+def cinema_visit(
+        customers: list[Customer],
+        hall_number: int,
+        cleaner: str[Cleaner.__name__],
```
Comment (viktoria-rybenchuk) [inline app/main.py:12]:
```suggestion
        cleaner: str,
```

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by pavlejviki — APPROVED** [review]
Good job!

**Pull request closed without merging**